packages/arb-token-bridge-ui/src/components/TransactionsTable/TransactionsTableDepositRow.tsx

```diff
@@ -76,17 +76,21 @@ function DepositRowTime({ tx }: { tx: MergedTransaction }) {
     )
   }
 
-  return <span>{tx.resolvedAt || tx.createdAt || 'N/A'}</span>
+  return (
+    <span className="whitespace-nowrap">
+      {tx.resolvedAt || tx.createdAt || 'N/A'}
+    </span>
+  )
 }
 
 function DepositRowTxID({ tx }: { tx: MergedTransaction }) {
   const { l1, l2 } = useNetworksAndSigners()
   const l2TxHash = tx.l1ToL2MsgData?.l2TxID
 
   return (
-    <div className="flex flex flex-col flex-col space-y-1">
-      <span className="text-dark">
-        L1:{' '}
+    <div className="flex flex-col space-y-1">
+      <span className="whitespace-nowrap text-dark">
+        Mainnet:{' '}
```

Comment:
<img width="662" alt="image" src="https://user-images.githubusercontent.com/13184582/216056783-47c56c41-d934-4598-9124-a038ce5d6a39.png">
inconsistent with the withdrawal row

Reply:
Very valid point. Thanks! 
Done.